src/main/webapp/scripts/suggest-script.js

```diff
@@ -0,0 +1,13 @@
+/* eslint-disable */
```

Comment:
Instead of disabling the linter for this file, you could set the element to have an onclick in this javascript.  So instead of setting the onclick in the html, when the page loads you can find the element in this javascript file and then add the onclick, as shown in the implementation in Javascript (https://www.w3schools.com/jsref/event_onclick.asp).

Reply:
Removed eslint-disable, changed to onclick similar to link